browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -85,6 +92,33 @@ void EphemeralStorageTabHelper::ReadyToCommitNavigation(
   CreateEphemeralStorageAreasForDomainAndURL(new_domain, new_url);
 }
 
+void EphemeralStorageTabHelper::ClearEphemeralLifetimeKeepalive(
+    const content::TLDEphemeralLifetimeKey& key) {
+  ClearLocalStorageKeepAlive(
```

Comment:
this would be simpler if TLDEphemeralLifetime owned `local_storage_namespace_`, but I'd rather do that refactoring in a follow-up